### PR TITLE
Remove S3 deployment

### PR DESCRIPTION
**Pull request by OzzieOrca**
Since this is all on Amplify, it should be safe to delete these buckets after:
https://s3.console.aws.amazon.com/s3/buckets/cru-missionhub-web-prod/?region=us-east-1
https://s3.console.aws.amazon.com/s3/buckets/cru-missionhub-web-stage/?region=us-east-1

### CODE REVIEW

**Comment by soberstadt** [comment]
@OzzieOrca sorry, I had added the review comment yesterday but didn't submit it :(

File: .travis.yml [hunk 1]
```diff
@@ -10,20 +10,16 @@ addons:
 
 script:
     - yarn test
-    - yarn global add apollo
-    - yarn gql:schema
-    - yarn gql:codegen
-    - yarn test:react:coverage
+    # Disable react tooling as types have been removed from the MissionHub schema
```
[the remark this answers is not in the document]
Comment (OzzieOrca) [reply .travis.yml:13]: If we ever pull the React dashboard out, I'd want this running in that repo. But I suppose we can figure out that tooling again later.

[the remark this answers is not in the document]
Comment (soberstadt) [reply .travis.yml:13]: Ok, I usually prefer deleting the code and going back to the git history to re-add it. But I don't feel strongly about it, so I'll approve this PR and let you decide.

[the remark this answers is not in the document]
Comment (OzzieOrca) [reply .travis.yml:13]: I'll clean it up in https://jira.cru.org/browse/MHP-3683

**Pull request merged**